### ISSUE TITLE
Pull data on Cash Sweep

### ISSUE THREAD

**Issue by hetkpatel** [jmfernandes/robin_stocks#371]
Is there a method to pull historical data on interest earned through the RH Cash Sweep program? Initially, I tried drawing a list of all transactions on the account and filtering for only Cash Sweep events...no luck.

Appreciate all the help I get.

**Comment by DarishSakeesing** [comment]
Yes, I would also like to get information on cash sweep and interests received
